lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -481,6 +496,14 @@ sub _compose_job_overview_search_args ($c) {
     return (\%search_args, \@groups);
 }
 
+sub _match_group ($regexes, $group) {
+    my $name = $group->name;
+    for my $regex (@$regexes) {
+        return 1 if $name =~ $regex;
+    }
+    return 0;
```

Comment:
how about https://perldoc.perl.org/List::Util#any

```suggestion
    return any { $name =~ $regex } @$regexes;
```

Reply:
Loops are faster than `List::Util` functions usually, but that's not really a factor here. So i'll change it to the shorter version.